RecoLocalTracker/SiStripZeroSuppression/BuildFile.xml

```diff
@@ -8,6 +8,7 @@
 <use name="CondFormats/SiStripObjects"/>
 <use name="CalibFormats/SiStripObjects"/>
 <use name="CalibTracker/Records"/>
+<use name="Geometry/TrackerGeometryBuilder"/>
```

Comment:
why was this needed?
There are no includes of `Geometry/TrackerGeometryBuilder` in ` RecoLocalTracker/SiStripZeroSuppression/`

Reply:
nevermind, I removed the comment; there's no problem here